perun-openapi/openapi.yml

```diff
@@ -2384,6 +2400,127 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/authzResolver/getGroupRoleNames:
```

Comment:
Although it is now written in the documentation, this method takes a `group` param of type int.

Reply:
I looked into documentation and as you can see on the printscreen, there are no parameters in this method.
![image](https://user-images.githubusercontent.com/52922281/84766149-9c62cb00-afd0-11ea-8ffe-54ad82b22235.png)

Reply:
The documentation is wrong, look at the implementation: https://github.com/CESNET/perun/blob/master/perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AuthzResolverMethod.java#L77